### DISCUSSION TITLE
The problem with AV on Chromie Craft

### DISCUSSION THREAD

**Discussion by Vlaxitov** [chromiecraft/chromiecraft#3577] [Suggestions]
The problem with AV on this server is with the tiny amount of players required to make it pop you only need one or two people back capping to make every game a daunting turtle. So that's what AV weekends are now, turtle after turtle after turtle after turtle, and the BG weekend I used to most look forward to has gotten to the point that some people will just stop queuing. If AV on this server was more blizzlike in the amount of players you needed to start a game back capping and forcing turtles wouldn't work.

Now I'm sure there's a handful of players on this server who would strongly object to that being a problem, mostly because they no longer have any real incentive to complete objectives and are just HK farming. I'd argue that for the new player or someone playing a new character it makes for a miserable experience which negatively impacts the overall health of the server in it's current iteration.

So whats the concluding suggestion here? Put a reasonable time limit on AV matches that encourages people to actually do the objectives. I also remember daily diminishing returns with getting honor kills on specific players during the time that AV was most relevant.

**Comment by DarylBlake21** [comment]
You have to compare 2 kinds of players: the ones farming reputation and the ones that want to do some pvp and actually want to enjoy Alterac Valley. If you want a quick BG then join Arathi Basin or Warsong Gulch. And talking about turtling, turtling is what actually makes AV alive, not just some braindead rush to boss. You join Alterac Valley and most of the games people avoid the PvP with the exception of a few players defending the boss. And there comes another problem, after they finish the reputation grind, they're not coming back. You keep seeing less people on every game.

**Reply by Vlaxitov** [reply]
I'd argue that there's a difference between a complete competitive game and a turtle, that you don't "actually want to enjoy AV" if you're not interested in doing the objectives of the game. Again, when it peaks and its 40 vs 40 you can have defense on either side with one side or the other completing the objectives anyway 9.9/10 times. Its only when the player count is low the turtling gets miserable. To put it into perspective, imagine wsg with an hour time limit or more and one side just controls mid to ensure that flag caps never happen. That's what AV is like with a low amount of players queuing up for it.

**Comment by plasma-93** [comment]
Some people just want to watch the world burn, they PvP in AV just to annoy people.

Some people pvp in AV because they want to PvP and AV is what's popping.

Some rush bosses and objectives because that's objectively the best way to play AV.

Some rush bosses and objectives because that's how you get rep the fastest.

There's no real fix for this without modifying AV, dunno what way you'd do that, rep for player kills, match limit of 30min, bigger bonuses for objectives, not sure, it's currently the least enjoyable part of the server as a whole so I would really like something to be done, Today before queuing I ate some edibles so that I wouldn't be entirely conscious while grinding rep on my alt, legit abusing substances to be less miserable lmao
Either way I'm the point in which I put people on ignore if they are turtling, there's nothing intelligent they can bring to any discussion anyhow.

**Comment by dennots** [comment]
State of AV on Chromie:
- When games go fast, people keep queing and AV dies late.
- When turtling happens, people stop queing and AV dies early.
- On a 15v15 setting, its easy for 2 or 3 people to provoke a turtle which isn't the case on 40v40.
- The argument of PvPers turtling is that they want the HKs for the achievement.
- The argument of PvEers against turtling is that they want the rep for rewards and never come back.
- The people farming just for the PvE rewards don't like having to do it while the PvP'ers feed from the misery of the PvE'ers. 

I don't know if there is a solution, but it is true that the BG was not designed to be run by so few as OP stated.

**Comment by DarylBlake21** [comment]
It's not just misery of the PvE'ers, you end up winning more HK on Warsong Gulch and Arathi Basin. The same as one losses his weekend farming reputation for rewards, the other ones can't do anything relevant during AV weekend. Few games give 150 hk, you usually end up with 5-10hk. Also there comes the other problem, the people turtling usually have better gear compared to the other ones farming reputation. Tbh i don't think there's a possible solution unless they put back rewards back to exalted or give them for free without any rep requirement. People are doing Alterac Valley because they need to gear up their alts, not because they enjoy PvP.

**Comment by Talthos** [comment]
I recall AV had repeatable turn-ins that would eventually trigger scripted NPC events (one of them would summon a 'Forest Lord' to the middle or something). Managing to trigger them made for more interesting games, imho, back before the rush strategy became meta later on.

Are those functional on ChromieCraft? I haven't had the chance to test it myself yet.
